website/screens/components/file-input/code/FileInputCodePage.tsx

```diff
@@ -0,0 +1,187 @@
+import { DxcStack, DxcTable, DxcLink } from "@dxc-technology/halstack-react";
+import QuickNavContainerLayout from "@/common/QuickNavContainerLayout";
+import QuickNavContainer from "@/common/QuickNavContainer";
+import Code from "@/common/Code";
+import DocFooter from "@/common/DocFooter";
+import Example from "@/common/example/Example";
+import modesFileInput from "./examples/ModesFileInput";
+
+const sections = [
+  {
+    title: "Props",
+    content: (
+      <DxcTable>
```

Comment:
Use the Code component to highlight default values.

Reply:
Done